src/os/user/iterate_cgo_test.go

```diff
@@ -0,0 +1,77 @@
+//go:build (aix || dragonfly || (!android && linux) || solaris) && cgo && !osusergo
+// +build aix dragonfly !android,linux solaris
+// +build cgo
+// +build !osusergo
+
+package user
+
```

Comment:
comment (somewhere) that this file is for unix systems that support fgetpwent/fgetgrent.